common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -419,7 +419,8 @@ def build_with_contentstore(self, contentstore, **kwargs):
         """
         names, generators = list(zip(*self.store_builders))
 
-        with nested(*(gen.build_with_contentstore(contentstore, **kwargs) for gen in generators)) as modulestores:
```

Comment:
I've been using Python for over a decade and it's only with this that I realize you can splat a generator. o.O Glad to see this line gone.